modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,537 @@
+
+With this update the extra padding was removed and separator lines in {productname} 6.2 multi-line toolbars are drawn correctly.
+
+=== Fixed a regression where pressing Enter added or deleted content outside the selection
+
+A regression found in {productname} 6.0 caused inline elements with `contenteditable="false"` attributes to be incorrectly removed when *Enter* or *Return* were pressed on a selection which began before, and entered into, the block containing the element.
+
+This regression was fixed in {productname} 6.2 and pressing *Enter* or *Return* in this circumstance no longer incorrectly removes these elements.
+
+=== Fixed a regression where pressing Enter deleted selected `contenteditable="false" <pre>` elements
+
+A regression found in {productname} 6.0 caused `preformatted` elements with `contenteditable="false"` attributes to incorrectly insert a newline when *Enter* or *Return* were pressed.
+
+This regression was fixed in {productname} 6.2 and pressing *Enter* or *Return* no longer inserts an erroneous newline.
+
+
+=== The `editor.insertContent()` API did not respect the `no_events` argument
+
+If, for example, `editor.insertContent('<p>test</p>', { no_events: true });` was run, `BeforeGetContent` and `SetContent` events were still fired, despite `no_events` being set to `true`.
+
+With this update, the API respects the `no_events: true` setting. Adding content to a {productname} document with `no_events` set to true now adds the content without firing events, as expected.
+
+[[deprecated]]
+== Deprecated
+
+=== The `autocompleter` configuration property, `ch`, has been deprecated
+
+The `ch` autocompleter configuration property will be removed in the next major release.
+
+Use the `trigger` property instead.
+
+[source,js]
+----
+trigger: '<string>',
+----
+
+
+[[known-issues]]
+== Known issues
+
+This section describes issues that users of {productname} 6.2 may encounter and possible workarounds for these issues.
+
+There are three known issues in {productname} 6.2, all affecting the new premium plugin, xref:mergetags.adoc[Merge Tags].
+
+=== Merge Tags cannot be manually added to a {productname} document in some circumstances
+
+There are several circumstances where typing the Merge tags prefix (by default, `+{{+`) does not activate the `autocompleter`. Consequently, choosing a Merge tag from the expected pop-up menu is not possible because the pop-up menu does not present.
+
+Moreover, in the circumstances where the `autocompleter` does not activate, even typing the entire Merge tag does not result in the string being turned in to a Merge tag.
+
+There is a workaround, however.
+
+If typing the Merge tags prefix does not work as expected, clicking the Merge tags toolbar button and selecting the desired Merge tag from the resultant menu inserts a functioning Merge tag.
```

Comment:
```suggestion
If typing the Merge tags prefix does not work as expected, clicking the Merge tags toolbar button or menu item and selecting the desired Merge tag from the resultant menu inserts a functioning Merge tag.
```
Maybe? It's just you can use either the toolbar or menu item IIUC.

Reply:
(discussion): We'll leave this for now. The main goal was to demonstrate that there were other ways to insert, but it's hard to get the wording right.